lib/cc/engine/duplication.rb

```diff
@@ -30,6 +30,10 @@ def initialize(directory:, engine_config:, io:)
       def run
         Dir.chdir(directory) do
           languages_to_analyze.each do |language|
+            if engine_config.debug?
+              $stderr.puts("Analyzing language #{language}")
```

Comment:
This seems to be the only log line that can't use `Reporter#debug` and has to duplicate the logic. Thoughts on just ditching it? Seems low value to me.